sound/mocp/Makefile

```diff
@@ -29,7 +29,7 @@ include $(INCLUDE_DIR)/nls.mk
 define Package/moc
   SECTION:=sound
   CATEGORY:=Sound
-  DEPENDS:=+libcurl +BUILD_PATENTED:libmad +libvorbis $(ICONV_DEPENDS) +alsa-lib +libid3tag +libflac +libsamplerate +libncursesw +libffmpeg +libltdl +libmagic +faad2 +libdb47
```

Comment:
You're looking at the wrong file, (right side is the new one).
https://github.com/diizzyy/packages/blob/08e74f346afa8b65c1661d531e25d3a3334a6d3c/sound/mocp/Makefile